src/commands/aem/rde/status.js

```diff
@@ -29,12 +31,11 @@ class StatusCommand extends BaseCommand {
 
   async printAsText() {
     try {
-      cli.log(`Info for cm-p${this._programId}-e${this._environmentId}`);
```

Comment:
We might want to keep this logger since I think some people are automatically checking the output of this command (not sure if they're checking it on text or json format) and therefore we might break them

Reply:
a more generic logger is shown now for all commands automatically, i.e.:
`Running StatusCommand on cm-p84002-e229103 (cod18017-rde-development - cod18017-rde-development-10-rde)`
People that are checking with some logic should use the json format as this is independent. We should break them if they do not ;-)

Reply:
outcome of the rde standup meeting today: add the three log lines back (status, reset, restart) and keep them additionally to the new generic logger.
the generic logger shall remain, with slight adjustments to the message, tbd.